contractcourt/htlc_success_resolver.go

```diff
@@ -86,6 +87,31 @@ func (h *htlcSuccessResolver) ResolverKey() []byte {
 	return key[:]
 }
 
+// waitForSpend waits for the given outpoint to be spent, and returns the
+// details of the spending tx.
+func (h *htlcSuccessResolver) waitForSpend(op *wire.OutPoint,
```

Comment:
seeing that the previous commit decoupled `waitForHeight` from the resolver, shouldn't we do the same here?

Reply:
Done!